store/sqlstore/user_store.go

```diff
@@ -1942,35 +1943,36 @@ func (us SqlUserStore) DemoteUserToGuest(userID string) (*model.User, error) {
 }
 
 func (us SqlUserStore) AutocompleteUsersInChannel(teamId, channelId, term string, options *model.UserSearchOptions) (*model.UserAutocompleteInChannel, error) {
-	autocomplete := &model.UserAutocompleteInChannel{}
-	uchan := make(chan store.StoreResult, 1)
+	var wg sync.WaitGroup
+	wg.Add(2)
```

Comment:
nit: can we `wg.Add(1)` right before each `go func() { ... }`? I find this helps with the copy-paste problem of forgetting to increment/decrement this when changing the semantics.

Reply:
Good point, I'll change that